README.md

```diff
@@ -223,6 +223,12 @@ Run it from your Android device:
 
 Tested on SM-930F
 
+Building for ARM
+----------------
+
+To build on ARM (such as a Raspberry Pi), certain ciphers must be disabled when running configure:
+
+   	 ./configure --disable-kex-lwe-frodo --disable-kex-mlwe-kyber --disable-kex-ntru --disable-aes-ni
```

Comment:
If autoconf can detect which platform configure is running on, then we could disable these automatically.